dags/openshift_nightlies/scripts/install/osde2e.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -ex
+
+while getopts v:a:j:o: flag
+do
+    case "${flag}" in
+        v) version=${OPTARG};;
+        j) json_file=${OPTARG};;
+        o) operation=${OPTARG};;
+        *) echo "ERROR: invalid parameter ${flag}" ;;
+    esac
+done
+
+_get_cluster_id(){
+    echo $(rosa list clusters -o json | jq -r '.[] | select(.name == '\"$1\"') | .id')
+}
+
+_download_kubeconfig(){
+    ocm get /api/clusters_mgmt/v1/clusters/$1/credentials | jq -r .kubeconfig > $2
+}
+
+_get_cluster_status(){
+    echo $(rosa list clusters -o json | jq -r '.[] | select(.name == '\"$1\"') | .status.state')
+}
+
+setup(){
+    mkdir /home/airflow/workspace
+    cd /home/airflow/workspace
+    export PATH=$PATH:/usr/bin
+    export AWS_REGION=us-west-2
+    export AWS_ACCESS_KEY_ID=$(cat ${json_file} | jq -r .aws_access_key_id)
+    export AWS_SECRET_ACCESS_KEY=$(cat ${json_file} | jq -r .aws_secret_access_key)
+    export NETWORK_TYPE=$(cat ${json_file} | jq -r .openshift_network_type)
+    export ROSA_ENVIRONMENT=$(cat ${json_file} | jq -r .rosa_environment)
+    export OCM_ENVIRONMENT=$(cat ${json_file} | jq -r .ocm_environment)
+    export ROSA_TOKEN=$(cat ${json_file} | jq -r .rosa_token_${ROSA_ENVIRONMENT})
+    export CLUSTER_NAME=$(cat ${json_file} | jq -r .openshift_cluster_name)
+    rosa login --env=${ROSA_ENVIRONMENT}
+}
+
+install(){
+    cat << EOF > /home/airflow/workspace/config.yaml
+cloudProvider:
+  providerId: aws
+  region: ${AWS_REGION}
+cluster:
+  multiAZ: true
+  name: ${CLUSTER_NAME}
+ocm:
+  token: "${ROSA_TOKEN}"
+rosa:
+  awsAccessKey: ${AWS_ACCESS_KEY_ID}
```

Comment:
We also need the ability to set the worker count and worker type. ie:
```
rosa:  
  computeNodes: 30
  computeMachineType: m5.2xlarge
```

Reply:
Added